tidy3d/components/data/data_array.py

```diff
@@ -1179,20 +1179,48 @@ class EMEModeIndexDataArray(DataArray):
     _data_attrs = {"long_name": "Propagation index"}
 
 
-class ChargeDataArray(DataArray):
+class ChargeCarrierDataArray(DataArray):
     """Charge data array.
 
     Example
     -------
     >>> n = [0, 1e-12, 2e-12]
     >>> p = [0, 3e-12, 4e-12]
-    >>> td = ChargeDataArray((1+1j) * np.random.random((3,3)), coords=dict(n=n, p=p))
+    >>> td = ChargeCarrierDataArray((1+1j) * np.random.random((3,3)), coords=dict(n=n, p=p))
     """
 
     __slots__ = ()
     _dims = ("n", "p")
 
 
+class ElectroStaticDataArray(DataArray):
+    # TODO proper docs Given the combinatorial amount of variations of electro-static performance parameters,
+    #  I think it makes sense to have a single data array that can handle these type of parameter sweeps, and we just
+    #  check for them. Note we need to validate the data aligns properly between the swept variaable. It might be the
+    #  case C-I-V all get swept at the same time
+    """
+    Semiconductor I-V curve data array.
+
+    Example
+    -------
+    >>> I = [0. 0, 1, 4]
+    >>> V = [-1, -0.5, 0, 0.5]
+    >>> td = ElectroStaticDataArray(data=I, coords={"Voltage (V)": V})
+
+    Semiconductor DC capacitance variations with respect to voltage.
+
+    Example
+    -------
+    >>> C = [0. 0, 1, 4]
+    >>> V = [-1, -0.5, 0, 0.5]
+    >>> td = ElectroStaticDataArray(data=C, coords={"Voltage (V)": V})
+    """
+
+    __slots__ = ()
+    _dims = ("Voltage (V)",)
+    _data_attrs = {"long_name": "Current (A)"}
```

Comment:
The name has been generalized but this will print everything as I-V curve since ` _data_attrs = {"long_name": "Current (A)"}`